src/compass/s1_geocode_slc.py

```diff
@@ -175,10 +175,11 @@ def run(cfg: GeoRunConfig):
         # Save burst corrections and metadata with new h5py File instance
         # because io.Raster things
         with h5py.File(output_hdf5, 'a') as geo_burst_h5:
+            root_path = '/'
             root_group = geo_burst_h5[root_path]
             identity_to_h5group(root_group, burst, cfg)
 
-            cslc_group = geo_burst_h5.require_group(f'{root_path}/CSLC')
+            cslc_group = geo_burst_h5.require_group('/CSLC')
```

Comment:
```suggestion
            cslc_group = geo_burst_h5.require_group(ROOT_PATH)
```
Can import from `h5_helpers.py`